integration-tests/testutils/config/config_test.go

```diff
@@ -48,7 +48,7 @@ func testConfigStruct(fileName string) *Config {
 	return &Config{
 		fileName,
 		"testrelease", "testchannel",
-		true, true, true, true}
+		true, true, true, true, true}
```

Comment:
Totally unrelated to this actual branch, sorry for this: could we use the field names in the struct init to make it easier to see what each value means? Something like:
```
return &Config{FileName: fileName, Release: "testrelease", Channel: "testchannel", RemoteTestbed: true, ...
```